[rospy.client][INFO] 2017-10-18 07:48:28,880: init_node, name[/exec], pid[30798]
[xmlrpc][INFO] 2017-10-18 07:48:28,884: XML-RPC server binding to 0.0.0.0:0
[rospy.init][INFO] 2017-10-18 07:48:28,891: ROS Slave URI: [http://10.0.0.116:38610/]
[xmlrpc][INFO] 2017-10-18 07:48:28,891: Started XML-RPC server [http://10.0.0.116:38610/]
[rospy.impl.masterslave][INFO] 2017-10-18 07:48:28,892: _ready: http://10.0.0.116:38610/
[xmlrpc][INFO] 2017-10-18 07:48:28,893: xml rpc node: starting XML-RPC server
[rospy.registration][INFO] 2017-10-18 07:48:28,902: Registering with master node http://10.0.0.116:11311
[rospy.registration][INFO] 2017-10-18 07:48:28,903: Registering publisher topic [/play] type [std_msgs/String] with master
[rospy.init][INFO] 2017-10-18 07:48:28,992: registered with master
[rospy.rosout][INFO] 2017-10-18 07:48:28,992: initializing /rosout core topic
[rospy.rosout][INFO] 2017-10-18 07:48:29,019: connected to core topic /rosout
[rospy.simtime][INFO] 2017-10-18 07:48:29,056: /use_sim_time is not set, will not subscribe to simulated time [/clock] topic
[rospy.internal][INFO] 2017-10-18 07:48:29,232: topic[/rosout] adding connection to [/rosout], count 0
[rospy.internal][INFO] 2017-10-18 07:48:30,648: topic[/play] adding connection to [/chatty], count 0
[rospy.internal][INFO] 2017-10-18 07:48:31,186: topic[/mavros/setpoint_raw/target_local] adding connection to [http://10.0.0.116:37215/], count 0
[rospy.internal][INFO] 2017-10-18 07:48:31,263: topic[/mavros/state] adding connection to [http://10.0.0.116:37215/], count 0
[rospy.internal][INFO] 2017-10-18 07:48:31,326: topic[/mavros/battery] adding connection to [http://10.0.0.116:37215/], count 0
[rospy.internal][INFO] 2017-10-18 07:48:31,334: topic[/mavros/local_position/pose] adding connection to [http://10.0.0.116:37215/], count 0
[rospy.internal][INFO] 2017-10-18 07:48:31,341: topic[/mavlink/from] adding connection to [http://10.0.0.116:37215/], count 0
[rospy.internal][INFO] 2017-10-18 07:48:31,686: topic[/mavros/mission/waypoints] adding connection to [http://10.0.0.116:37215/], count 0
[rospy.internal][INFO] 2017-10-18 07:48:31,796: topic[/mavros/rc/override] adding connection to [/mavros], count 0
[rospy.internal][INFO] 2017-10-18 07:48:31,916: topic[/state_command] adding connection to [/state_driving_to_cone], count 0
[rosout][INFO] 2017-10-18 07:48:31,927: Exec is starting
[rospy.internal][INFO] 2017-10-18 07:48:31,950: topic[/exec_command] adding connection to [http://10.0.0.116:39812/], count 0
[rospy.internal][INFO] 2017-10-18 07:48:31,961: topic[/exec_command] adding connection to [http://10.0.0.116:43297/], count 0
[rospy.internal][INFO] 2017-10-18 07:48:31,981: topic[/exec_command] adding connection to [http://10.0.0.116:38610/], count 2
[rospy.internal][INFO] 2017-10-18 07:48:31,969: topic[/exec_command] adding connection to [http://10.0.0.116:41980/], count 1
[rospy.internal][INFO] 2017-10-18 07:48:32,003: topic[/exec_command] adding connection to [http://10.0.0.116:35502/], count 4
[rospy.internal][INFO] 2017-10-18 07:48:32,021: topic[/state_command] adding connection to [/state_driving_away], count 1
[rospy.internal][INFO] 2017-10-18 07:48:32,025: topic[/state_command] adding connection to [/state_follow_waypoint], count 2
[rospy.internal][INFO] 2017-10-18 07:48:32,027: topic[/exec_command] adding connection to [/exec], count 0
[rospy.internal][INFO] 2017-10-18 07:48:32,037: topic[/state_command] adding connection to [/state_avoiding_obstacle], count 3
[rosout][INFO] 2017-10-18 07:48:36,928: New parameters received
[rosout][INFO] 2017-10-18 07:48:43,466: /exec MsgId 46 detected. WP item # 7936
[rosout][INFO] 2017-10-18 07:48:44,927: New parameters received
[rosout][INFO] 2017-10-18 07:48:45,928: New parameters received
[rosout][INFO] 2017-10-18 07:48:50,928: New parameters received
[rosout][INFO] 2017-10-18 07:48:51,928: New parameters received
[rosout][INFO] 2017-10-18 07:48:52,928: New parameters received
[rosout][INFO] 2017-10-18 07:48:53,928: New parameters received
[rosout][INFO] 2017-10-18 07:48:54,928: New parameters received
[rospy.core][INFO] 2017-10-18 07:48:59,863: signal_shutdown [signal-2]
[rospy.internal][INFO] 2017-10-18 07:48:59,895: topic[/play] removing connection to /chatty
[rospy.internal][ERROR] 2017-10-18 07:49:00,067: Unable to initiate TCP/IP socket to 10.0.0.116:44721 (http://10.0.0.116:39812/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2017-10-18 07:49:00,126: topic[/state_command] removing connection to /state_follow_waypoint
[rospy.internal][ERROR] 2017-10-18 07:49:00,195: Unable to initiate TCP/IP socket to 10.0.0.116:39697 (http://10.0.0.116:35502/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][ERROR] 2017-10-18 07:49:00,210: Unable to initiate TCP/IP socket to 10.0.0.116:44738 (http://10.0.0.116:41980/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2017-10-18 07:49:00,213: topic[/state_command] removing connection to /state_driving_to_cone
[rospy.internal][INFO] 2017-10-18 07:49:00,249: topic[/state_command] removing connection to /state_driving_away
[rospy.internal][ERROR] 2017-10-18 07:49:00,265: Unable to initiate TCP/IP socket to 10.0.0.116:42386 (http://10.0.0.116:43297/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2017-10-18 07:49:00,298: topic[/mavros/rc/override] removing connection to /mavros
[rospy.internal][ERROR] 2017-10-18 07:49:00,301: Unable to initiate TCP/IP socket to 10.0.0.116:45499 (http://10.0.0.116:38610/): Traceback (most recent call last):
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 557, in connect
    self.read_header()
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 650, in read_header
    self._validate_header(read_ros_handshake_header(sock, self.read_buff, self.protocol.buff_size))
  File "/opt/ros/kinetic/lib/python2.7/dist-packages/rospy/impl/tcpros_base.py", line 593, in _validate_header
    raise TransportInitError("remote error reported: %s"%header['error'])
TransportInitError: remote error reported: node shutting down

[rospy.internal][INFO] 2017-10-18 07:49:00,302: topic[/exec_command] removing connection to /exec
[rospy.internal][INFO] 2017-10-18 07:49:00,305: topic[/rosout] removing connection to /rosout
[rospy.internal][INFO] 2017-10-18 07:49:00,312: topic[/state_command] removing connection to /state_avoiding_obstacle
[rospy.internal][INFO] 2017-10-18 07:49:00,317: topic[/mavros/mission/waypoints] removing connection to http://10.0.0.116:37215/
[rospy.internal][INFO] 2017-10-18 07:49:00,317: topic[/exec_command] removing connection to http://10.0.0.116:43297/
[rospy.internal][INFO] 2017-10-18 07:49:00,317: topic[/exec_command] removing connection to http://10.0.0.116:39812/
[rospy.internal][INFO] 2017-10-18 07:49:00,318: topic[/exec_command] removing connection to http://10.0.0.116:38610/
[rospy.internal][INFO] 2017-10-18 07:49:00,318: topic[/exec_command] removing connection to http://10.0.0.116:41980/
[rospy.internal][INFO] 2017-10-18 07:49:00,319: topic[/exec_command] removing connection to http://10.0.0.116:35502/
[rospy.internal][INFO] 2017-10-18 07:49:00,321: topic[/mavros/state] removing connection to http://10.0.0.116:37215/
[rospy.internal][INFO] 2017-10-18 07:49:00,327: topic[/mavlink/from] removing connection to http://10.0.0.116:37215/
[rospy.internal][INFO] 2017-10-18 07:49:00,330: topic[/mavros/local_position/pose] removing connection to http://10.0.0.116:37215/
[rospy.internal][INFO] 2017-10-18 07:49:00,333: topic[/mavros/battery] removing connection to http://10.0.0.116:37215/
[rospy.internal][INFO] 2017-10-18 07:49:00,334: topic[/mavros/setpoint_raw/target_local] removing connection to http://10.0.0.116:37215/
[rospy.impl.masterslave][INFO] 2017-10-18 07:49:00,334: signal-2
[rospy.core][INFO] 2017-10-18 07:49:00,873: signal_shutdown [atexit]
